.github/dependabot.yml

```diff
@@ -0,0 +1,14 @@
+version: 2
```

Comment:
Whats the effect of this?

Reply:
You mean the config-file or [dependabot](https://github.com/dependabot) itself?
Some time ago, dependabot was an externally managed bot, that creates PRs for updating the (maven/docker/gh-action/whatever) dependencies of repositories.
Meanwhile, dependabot is a Github build-in feature that can be enabled by this configuration.
Every day it checks the listed dependencies and creates PRs for each update available.